vpc-sample/README.md

```diff
@@ -0,0 +1,31 @@
+# Using egress and ingress settings to restrict access to services 
+
+A Google Cloud Project is required in order to run the sample. The project should have the following API's enabled:
+
+* Cloud Run
+* Cloud Functions
+* Serverless VPC Access API 
+
+## Deploy the Function 
+
+From inside the vpc-sample directory:
+
+`gcloud functions deploy restricted-function --runtime=python38 --trigger-http --no-allow-unauthenticated --ingress-settings=internal-only --entry-point=hello_world`
```

Comment:
Nit: use triple backticks to create cleaner blocks, with the `sh` marker for code formatting.

```sh
gcloud functions deploy restricted-function --runtime=python38 --trigger-http --no-allow-unauthenticated --ingress-settings=internal-only --entry-point=hello_world
```